scripts/cpplint.py

```diff
@@ -281,7 +282,12 @@
 # flag. By default all errors are on, so only add here categories that should be
 # off by default (i.e., categories that must be enabled by the --filter= flags).
 # All entries here should start with a '-' or '+', as in the --filter= flag.
-_DEFAULT_FILTERS = ['-build/include_alpha']
+_DEFAULT_FILTERS = [
+    '-build/include_alpha',
+    '-whitespace/',
+    '-readability/braces',
+    '-readability/identifier_spacing',
```

Comment:
None of the above change to the extent that cpplint would even notice, as far as I am aware. Notably, my understanding of the Friday meeting was that braces stay as-is.

Reply:
At the same time, if clang-format is the source-of-truth for this, why bother verifying it with another tool?

Reply:
`cpplint` appears to be highly portable, and indeed very easy to set up. The same isn't necessarily true for `clang-format`. We might do this once we have built a lot of trust in `clang-format`. But why do it now?

Reply:
This PR enables clang-format on Travis, so it doesn't *need* to be portable. (I mean, it *is* portable, and there's packages on most distributions for it.) The point is having a single source of truth for formatting rules. Having similar rules checked by two tools just allows for them to get out of sync, and at some point someone will end up asking whether to follow the linter or clang-format. Why not just make it clear from the outset that clang-format is the arbiter of style now?

Reply:
> This PR enables clang-format on Travis, so it doesn't need to be portable. (I mean, it is portable, and there's packages on most distributions for it.)

I can run `cpplint` on all my systems before pushing or even completing any commit. This isn't true of `clang-format`, unfortunately. (Yes, some of my systems don't have `clang-format` available.)

You do have a point in the split-brain problem, but I'd rather risk getting there than switching over without backup.

Reply:
OK, how about we re-enable these flags so that they are run by default locally, but disable them in Travis so that we don't ever get conflicting results from CI?

Reply:
Disable only the ones that clang-format would break; keep the rest.